pontoon/base/templates/locale_selector.html

```diff
@@ -60,5 +60,20 @@
       {% endfor %}
       <li class="no-match">No results</li>
     </ul>
+
+    {% if part %}
+    <div class="static-links">
+      <a href="{{ url('pontoon.teams') }}">
+        <div class="all-resources">
+          <span class="title">All Teams</span>
+        </div>
+      </a>
+      {% if locale %}<a href="{{ url('pontoon.locale', locale.code) }}">
+        <div class="all-resources">
```

Comment:
The css style for all-resources are same with all this static links, But I didn't thought about selectors, fixing now.